### PR TITLE
docs: Update JWT code example to latest codebase

### PR DESCRIPTION
**Pull request by ruymon**
Closes: #1510 

## ✅ Checklist

- [ x ] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [ x ] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ x ] I performed a functional test on my final commit

---

## Changelog

_[Short description of what has changed]_

Replaced the NextAuth Option code snippet to match the current boilerplate.

---

## Screenshots

N/A

_[Screenshots]_

💯

### CODE REVIEW

File: www/src/pages/en/usage/next-auth.mdx [hunk 1]
```diff
@@ -195,17 +195,17 @@ I.e.:
...
--     session({ session, user }) {
-+     session({ session, token }) {
--       if (session.user) {
-+       if (session.user && token.sub) {
--         session.user.id = user.id;
-+         session.user.id = token.sub;
-        }
-        return session;
-      }
+-     session: ({ session, token }) => ({
```
Comment (juliusmarminge) [inline www/src/pages/en/usage/next-auth.mdx:198]: user

**Review by juliusmarminge — CHANGES_REQUESTED** [review]

File: www/src/pages/en/usage/next-auth.mdx (hunk 1 above)
Comment (ruymon) [reply www/src/pages/en/usage/next-auth.mdx]: My bad 😕

Fixed.

**Comment by ruymon** [comment]
TBH, this snippet is also in the translated files. I'll also fix that throught the languages before. For now, I'll convert it back to draft.

**Comment by ruymon** [comment]
Converted to draft was a misclick...

**Review by c-ehrlich — APPROVED** [review]
thanks for the fix :) and sorry for taking forever to get it in

**Pull request merged**